### PR TITLE
Add docs for setting up Vim jsonschema.

### PR DESCRIPTION
**Pull request by cpuguy83**

### CODE REVIEW

File: website/docs/editor-support.md [hunk 1]
```diff
@@ -58,3 +58,53 @@ The yaml plugin will complain that it is an incorrect type. To fix this you can
...
+following vim command to install the correct LSP, this only needs to be done one
+time.
+
+```
+:LspInstallServer
+```
+
+Finally, in your project dir you can add a file `.vim-lsp-settings/settings.json`
+where we'll put the yaml LSP config for the particular project. Here is an example
+similar to the vscode example above
```
Comment (adamperlin) [inline website/docs/editor-support.md:90]:
```suggestion
similar to the vscode example above:
```

File: website/docs/editor-support.md [hunk 2]
```diff
@@ -58,3 +58,53 @@ The yaml plugin will complain that it is an incorrect type. To fix this you can
...
+		],
+		"completion": true,
+		"validate": true
+	}
+}
+```
+
+In the "filematch' section you can add the file patterns to associate the schema
+with.
+Replace `<version>` with the version of the schema that corresponds to the dalec
```
Comment (adamperlin) [inline website/docs/editor-support.md:109]:
```suggestion
In this case,  `<version>`  should be replaced with the specific Dalec frontend version  (i.e., `v0.6.1`) that is referenced in any files which match the `fileMatch` pattern. To pick up whatever the latest schema is, released or not, use `main`.
```

File: website/docs/editor-support.md [hunk 3]
```diff
@@ -58,3 +58,53 @@ The yaml plugin will complain that it is an incorrect type. To fix this you can
...
+				"url": "https://raw.githubusercontent.com/Azure/dalec/<version>/docs/spec.schema.json"
+			}
+		],
+		"completion": true,
+		"validate": true
+	}
+}
+```
+
+In the "filematch' section you can add the file patterns to associate the schema
```
Comment (adamperlin) [inline website/docs/editor-support.md:107]:
```suggestion
In the `"fileMatch"` section you can add the file patterns to associate the schema
```

**Comment by cpuguy83** [comment]
Applied all those suggestions, thanks!

File: website/docs/editor-support.md [hunk 4]
```diff
@@ -58,3 +58,53 @@ The yaml plugin will complain that it is an incorrect type. To fix this you can
...
+		"completion": true,
+		"validate": true
+	}
+}
+```
+
+In the `"fileMatch"` section you can add the file patterns to associate the schema
+with.
+In this case,  `<version>`  should be replaced with the specific Dalec frontend version  (i.e., `v0.6.1`) that is referenced in any files which match the `fileMatch` pattern. To pick up whatever the latest schema is, released or not, use `main`.
+used for the matched files.
```
Comment (adamperlin) [inline website/docs/editor-support.md:110]:
```suggestion
```

Comment (cpuguy83) [reply website/docs/editor-support.md]: Fixed.

**Comment by DannyBrito** [comment]
this was where the node `version` was a problem for Adam, right? if some maybe we could add a note about this test with node version >= X?

**Comment by cpuguy83** [comment]
> this was where the node version was a problem for Adam, right? if some maybe we could add a note about this test with node version >= X?

Added a note about this.

**Review by adamperlin — APPROVED** [review]
lgtm!

**Comment by cpuguy83** [comment]
@DannyBrito LGTY?

**Review by DannyBrito — APPROVED** [review]
lgtm

**Pull request merged**